File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/3 - 3 and 4 November 2018/0 - My Exam/01. Series Calculator .py
import math
name = input()
count_seasons = int(input())
count_episodes = int(input())
time_episode = float(input())

adverts = time_episode * 0.2
time_episode = time_episode + adverts
added_special_time = count_seasons * 10
total = (time_episode * count_episodes * count_seasons) + added_special_time

print(f"Total time needed to watch the {name} series is {math.floor(total)} minutes.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/04. Online Exam - 15 and 16 June 2019/01. Series Calculator.js
function seriesCalculator(params) {
    let name = params.shift();
    let seasons = Number(params.shift());
    let episodes = Number(params.shift());
    let time = Number(params.shift()) * 1.2;

    let minutes = Math.floor((seasons * episodes * time) + (seasons * 10)); 
    console.log(`Total time needed to watch the ${name} series is ${minutes} minutes.`);
}


seriesCalculator(['Lucifer', '3', '18', '55']);  // Expected Output:
// Total time needed to watch the Lucifer series is 3594 minutes.

seriesCalculator(['Game of Thrones', '7', '10', '50']);  // Expected Output:
// Total time needed to watch the Game of Thrones series is 4270 minutes.

seriesCalculator(['Riverdale', '3', '21', '45']);  // Expected Output:
// Total time needed to watch the Riverdale series is 3432 minutes.